grr/server/grr_response_server/gui/ui/lib/models/model_test_util.ts

```diff
@@ -96,3 +96,22 @@ export function newClientApproval(args: Partial<ClientApproval> = {}):
     ...args,
   };
 }
+
+/**
+ * A helper function to construct a FlowResultSet object.
```

Comment:
This JSDoc is unneeded - it adds no new information. For the whole file, we disabled the linter error of missing JSDoc in order to reduce the visual clutter. Please remove.